src/components/TweetView/Tweet.js

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import cx from 'classnames';
+import config from '../../config.js';
+import styles from './Tweet.scss';
+
+const Tweets = (props) => {
+  return (
+    <div className={styles.block}>
+      <div className={styles.image}>
+        <img width={config.avatarSize} height={config.avatarSize} src={props.image}/>
```

Comment:
can we add `alt` attribute? it's required for accessibility

possibly a better name for `image` will be `imageUrl`